### ISSUE TITLE
JSON or standard logs

### ISSUE THREAD

**Issue by antho1404** [mesg-foundation/engine#1823]
Currently, it looks like there is no way to force the logs in a more standard log system like json.

The orchestrator doesn't accept any other type of logs than the plain text and the validator doesn't seem to accept the `log_format = "json"` in the `config.toml`.

Current logs are not standard and really hard to parse.

**Comment by NicolasMahe** [comment]
Tendermint logger actually has a JSON and fmt version:
```go
log.NewTMLogger()
log.NewTMJSONLogger()
log.NewTMFmtLogger()
```
https://pkg.go.dev/github.com/tendermint/tendermint@v0.33.3/libs/log?tab=doc

I will try the fmt and JSON version to have something more standard.
